### PR TITLE
MT update

### PR DESCRIPTION
**Pull request by thast**
A few details:
- corrected somes typos
-add Refraction angle section

Suggested reviewers:

-@(lheagy)

Before merging: please run a spell check through your documents!

### CODE REVIEW

File: content/maxwell3_fdem/MT/MT_N_layered_Earth.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ We present here a 1D modelisation of the Magnetotelluric waves in a layered Eart
...
 
-Magnetotelluric is a widely used method, especially for imaging geothermal. Its ability to image deep structure, up to several kilometers depth, is unique in EM geophysics. It is a passive method that use plane waves generated mostly in the Earth's Atmosphere. High frequency waves are mainly produced by lightning strikes all around the globe, traveling through the Earth's Ionosphere that acts as a waveguide. Low frequency waves are produced through the interaction of the Earth's Ionosphere with solar wind and Earth's magnetic field.
+Magnetotelluric (MT) is a widely used method, especially for imaging geothermal. Its ability to image deep structure, up to several kilometers depth, is unique in EM geophysics. It is a passive method that use plane waves generated mostly in the Earth's Atmosphere. High frequency waves are mainly produced by lightning strikes all around the globe, traveling through the Earth's Ionosphere that acts as a waveguide. Low frequency waves are produced through the interaction of the Earth's Ionosphere with solar wind and Earth's magnetic field.
```
Comment (lheagy) [inline content/maxwell3_fdem/MT/MT_N_layered_Earth.rst]: 'Magnetotellurics (MT)' or 'The Magnetotelluric method'  

(just a tip, not required - if you split the lines CMD+ALT+Q in sublime, it makes versioning a bit more transparent. then each line is versioned independently)

File: content/maxwell3_fdem/MT/MT_N_layered_Earth.rst [hunk 2]
```diff
@@ -85,7 +85,7 @@ with k the wavenumber:
     :label: kwavenumber
 
 
-In the ground, we can generally assume that the displacement current is negligible, which means :math:`\sigma \ll \omega \varepsilon`. In this case 
+In the ground, we can generally assume that the displacement current is negligible, which means :math:`\sigma \gg \omega \varepsilon`. In this case
```
Comment (lheagy) [inline content/maxwell3_fdem/MT/MT_N_layered_Earth.rst]: good catch!

File: content/maxwell3_fdem/MT/MT_N_layered_Earth.rst [hunk 3]
```diff
@@ -241,11 +241,64 @@ These coefficients tell us how much energy of the incoming has been reflected or
...
+
+ Refraction and Reflection angles
+
+
 In reality, the incident wave is coming from all the possible directions in the air. So how valid is our assumption of an incident vertical wave?
 
 What is important is the refraction angle at the Air-Earth interface, the angle of the transmitted wave in the ground.
 
-As any wave, electromagnetic waves follow Snell's law
+As any wave, electromagnetic waves follow Snell's law, that we can derive from the Maxwell's equation.
```
Comment (lheagy) [inline content/maxwell3_fdem/MT/MT_N_layered_Earth.rst]: could link Snell's law to wikipedia if you like, and perhaps Maxwell's equations to `maxwells_equations_index`

File: content/maxwell3_fdem/MT/MT_N_layered_Earth.rst [hunk 4]
```diff
@@ -241,11 +241,64 @@ These coefficients tell us how much energy of the incoming has been reflected or
...
+.. math::
+    ||k_{ix}||=||k_{tx}||
+
+.. math::
+    ||k_{air}|| *sin \theta_i = ||k_{earth}||*sin \theta_t
+
+.. math::
+    \theta_t = sin^{-1} (\frac{||k_{air}||}{||k_{earth}||} *sin (\theta_i))  \simeq 0
+
+As :math:`\frac{||k_{air}||}{||k_{earth}||}` is a really small number as the conductivity of the earth is usually several order of magnitude higher than the one of the air, :math:`\theta_t \simeq 0`. The wave is transmitted vertically into the ground, whatever is the incident angle. Our data are then not affected by it.
```
Comment (lheagy) [inline content/maxwell3_fdem/MT/MT_N_layered_Earth.rst]: `regardless of incident angle` rather than `whatever is the incident angle`

**Comment by lheagy** [comment]
looks good! I left a few minor comments for you

**Comment by thast** [comment]
@lheagy 
Failed after merging master. Due to a dead external link. See caption below.

build 1446 - ubcgif_em - travis ci - google chrome 2016-05-25 11 56 52

**Comment by lheagy** [comment]
We can ask @lacmajedrez : it looks like this site (https://luzcaudillo.wordpress.com/) has been taken down. Is there one you would like to put in its place? or should we remove the link?

**Comment by thast** [comment]
It seems it is used for an attribution. One suggestion might be to replace it by her github profile

**Pull request merged**